src/utils/memoize.tsx

```diff
@@ -0,0 +1,33 @@
+export default function memoize<Result, Deps extends ReadonlyArray<any>>(
```

Comment:
haven't you consider adding it to https://github.com/wcandillon/react-native-redash? I'll happily see it there

Reply:
maybe it'll be better to add something similar to reanimated itself? or try to optimize it so this is not necessary